### PR TITLE
Add Play 2.7 smoke test via sbt build

### PR DESCRIPTION
**Pull request by bantonsson**

### CODE REVIEW

**Review by richardstartin — APPROVED** [review]
👍

**Review by mcculls — APPROVED** [review]

File: dd-smoke-tests/play-2.7/application/project/plugins.sbt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+val playVersion = System.getProperty("play.version", "2.8.2")
```
Comment (tylerbenson) [inline dd-smoke-tests/play-2.7/application/project/plugins.sbt:1]:
```suggestion
val playVersion = System.getProperty("play.version", "2.7.5")
```

(Assuming `playVersion` is correct.)

**Review by tylerbenson — APPROVED** [review]
One version issue...

File: dd-smoke-tests/play-2.7/application/project/plugins.sbt (hunk 1 above)
Comment (bantonsson) [reply dd-smoke-tests/play-2.7/application/project/plugins.sbt:1]: Thanks

**Pull request merged**